pkg/controller/util/endpoint/controller_utils_test.go

```diff
@@ -926,10 +926,10 @@ func TestEndpointSubsetsEqualIgnoreResourceVersion(t *testing.T) {
 			expected: false,
 		},
 		{
-			name:     "Pod ResourceVersion changed",
```

Comment:
same comment here about adding a test case and keeping this one as-is

Reply:
done, thanks.